tests/e2e/Services/Account/AccountCustomClientTest.php

```diff
@@ -744,8 +743,31 @@ public function testUpdateAccountEmailSearch($data): void
 
     public function testCreatePhone(): array
     {
-        $number = '+123456789';
+        $to = App::getEnv('_APP_MESSAGE_SMS_PROVIDER_MSG91_TO');
+        $from = App::getEnv('_APP_MESSAGE_SMS_PROVIDER_MSG91_FROM');
+        $authKey = App::getEnv('_APP_MESSAGE_SMS_PROVIDER_MSG91_AUTH_KEY');
+        $senderId = App::getEnv('_APP_MESSAGE_SMS_PROVIDER_MSG91_SENDER_ID');
 
+        if ($to === '' || $from === '' || $authKey === '' || $senderId === '') {
+            $this->markTestSkipped('SMS provider not configured');
+        }
+
+        $number = $to;
+        $response = $this->client->call(Client::METHOD_POST, '/messaging/providers/msg91', \array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey'],
+        ]), [
+            'providerId' => 'unique()',
```

Comment:
Let's use the `ID::unique()` function here

Reply:
done this